### PR TITLE
I hope robustin dies in a garbage fire

### PR DESCRIPTION
**Pull request by Iamgoofball**
there's nothing fun about this in the slightest

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
explain the meme

**Comment by kevinz000** [comment]
"High Functioning Zombie"
:joy:

**Comment by PKPenguin321** [comment]
Wait... I know this pr!

File: code/modules/mob/living/carbon/human/species_types/zombies.dm [hunk 1]
```diff
@@ -1,56 +0,0 @@
...
-	//  because it persists through species transitions
-	var/obj/item/organ/zombie_infection/infection
-	infection = C.getorganslot("zombie_infection")
-	if(!infection)
-		infection = new()
-		infection.Insert(C)
-
-
-// Your skin falls off
-/datum/species/krokodil_addict
```
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species_types/zombies.dm]: goof you made this one

**Comment by Cyberboss** [comment]
Freeze

**Pull request closed without merging**